### PR TITLE
post-conf changes

### LINKED ISSUE #1785: Add 2024 Conference Videos and Flickr photos to Conf. website
We have a list of the videos from the GraphQL Conference as well as the photos from the event 🎉

It looks like the videos get populated on the conf website from the _videos.ts file but it's not clear to me how those are getting pulled. I have all the video links, which are on a new YT channel.

@dimaMachina or @YassinEldeeb can either of you assist?

### PR DESCRIPTION
**Pull request by YassinEldeeb**
# GraphQLConf 2024 Post-Event Website Transition

#### Sessions Updates
- [x] Link videos for each talk and workshop

#### Hero page Updates

- [x] Remove Call to Action buttons:
  - [x] Get Your Ticket
  - [x] Join as a Sponsor
  - [x] Join as a Partner
- [x] Put up Post Event Message:
  - [x] Thank you for Attending!
  - [x] Message: "Thank you to all who joined us for GraphQLConf 2024! We look forward to seeing you at future events. To experience the best of this year's event, be sure to watch session recordings and slides from speakers, available on the event schedule for each talk."
  - [x] Add button: “Explore all recordings and slides”
- [x] Add Flickr album link: [Featured Photos](https://flic.kr/s/aHBqjBHPhr)
- [x] Hide Register button on top navigation bar
- [x] Hide Partner button on top navigation bar
- [x] Hide Pricing & Registration section (including Register Now button)
- [x] Hide What’s Included section
- [x] Hide Location & Venue section (including transportation, airport, parking info)
- [x] Update conference banner to invite people to watch the talks

#### FAQ Updates
- [x] Hide Emergency Resources section
- [x] Hide Onsite Resources section
- [x] Hide Visa Letter Request section

Fixes #1785

### CODE REVIEW

File: src/app/conf/2024/page.tsx [hunk 1]
```diff
@@ -75,21 +76,31 @@ export default function Page() {
...
-            >
-              Join as a Sponsor
-            </a>
-            <a
-              href="/conf/2024/partner#program"
-              className="whitespace-nowrap with-arrow w-40 hover:text-primary transition-colors"
-            >
-              Join as a Partner
-            </a>
+            <Button href="#attend">Explore all recordings and slides!</Button>
```
Comment (YassinEldeeb) [inline src/app/conf/2024/page.tsx]:
```suggestion
            <Button href="/schedule">Explore all recordings and slides!</Button>
```

**Comment by jorydotcom** [comment]
🎉  @benjie - Yassin's done it!

File: vercel.json [hunk 2]
```diff
@@ -5,6 +5,16 @@
...
+    {
+      "source": "/conf/gallery",
+      "destination": "https://www.flickr.com/photos/linuxfoundation/albums/72177720320386221/",
+      "permanent": true
+    },
+    {
+      "source": "/conf/gallery/",
+      "destination": "https://www.flickr.com/photos/linuxfoundation/albums/72177720320386221/",
+      "permanent": true
+    },
```
Comment (benjie) [inline vercel.json:17]: Should these be `/conf/2024/gallery` instead?

File: src/app/conf/2024/_videos.ts [hunk 3]
```diff
@@ -0,0 +1,280 @@
+export const videos = [
+  {
+    id: "b45lWgkVLYA",
```
Comment (benjie) [inline src/app/conf/2024/_videos.ts]: We need to link to the new channel's videos - @jorydotcom and I will action this :+1:

**Review by benjie — CHANGES_REQUESTED** [review]
Amazing; thanks for your work on this! Just a couple things before we can ship it :raised_hands:

**Comment by jemgillam** [comment]
I've opened a PR at https://github.com/YassinEldeeb/graphql.github.io/pull/3 for the updated video IDs

File: src/app/conf/2024/_videos.ts (hunk 3 above)
Comment (benjie) [reply src/app/conf/2024/_videos.ts]: Done; thanks @jemgillam

File: vercel.json (hunk 2 above)
Comment (YassinEldeeb) [reply vercel.json:17]: we have 2 galleries at the moment, one for 2023, and another for 2024, the default should always redirect to the latest conf, in that case 2024, so I think we should leave it as is `/conf/gallery`.

Comment (jorydotcom) [reply vercel.json:17]: /conf/gallery is 404'ing in the deploy preview, will the redirects work after deploy?

Comment (benjie) [reply vercel.json:17]: > the default should always redirect to the latest conf, in that case 2024, so I think we should leave it as is /conf/gallery.

Makes sense; but we should have a permalink of `/conf/2024/gallery` too IMO; at the moment we have:

https://graphql-github-io-git-fork-yassi-fb0c5c-the-graph-ql-foundation.vercel.app/conf/2024/gallery/ - 404

https://graphql-github-io-git-fork-yassi-fb0c5c-the-graph-ql-foundation.vercel.app/conf/gallery/ - redirects to Flickr 2024

Comment (YassinEldeeb) [reply vercel.json:17]: > /conf/gallery is 404'ing in the deploy preview, will the redirects work after deploy?

oh I'll fix it, it does seem to be broken indeed

Comment (YassinEldeeb) [reply vercel.json:17]: > Makes sense; but we should have a permalink of /conf/2024/gallery too IMO

yeah, agree! I'll add it

**Review by benjie — APPROVED** [review]
I think the gallery link is still broken, but I don't know if that magically fixes itself on deploy. I'll go ahead and ship this, we can fix the gallery link in a follow-up PR.

**Comment by benjie** [comment]
Thanks for all your hard work on this @YassinEldeeb - excited to get this out there!

**Pull request merged**